asistente_ladm_col/logic/ladm_col/data/ladm_data.py

```diff
@@ -203,19 +460,21 @@ def get_parcel_data_to_compare_changes(self, db, search_criterion=None, layer_mo
             dict_attrs = dict()
             for field in layers[self.names.OP_PARCEL_T][LAYER].fields():
                 if field.name() in parcel_fields_to_compare:
+                    key_value_parcel = mapping_parcels_field[field.name()] if field.name() in mapping_parcels_field else field.name()
+
                     if field.name() == self.names.OP_PARCEL_T_PARCEL_TYPE_F:
                         # Go for domain value, instead of t_id
                         value = self.get_domain_value_from_code(db, self.names.OP_CONDITION_PARCEL_TYPE_D, feature.attribute(field.name()))
                     else:
                         value = feature.attribute(field.name())
-                    dict_attrs[field.name()] = value
-
+                    dict_attrs[key_value_parcel] = value
             dict_attrs[self.names.T_ID_F] = feature[self.names.T_ID_F]
 
-            if dict_attrs[self.names.OP_PARCEL_T_PARCEL_NUMBER_F] in dict_features:
-                dict_features[dict_attrs[self.names.OP_PARCEL_T_PARCEL_NUMBER_F]].append(dict_attrs)
+            key_value_parcel_number = mapping_parcels_field[self.names.OP_PARCEL_T_PARCEL_NUMBER_F] if self.names.OP_PARCEL_T_PARCEL_NUMBER_F in mapping_parcels_field else self.names.OP_PARCEL_T_PARCEL_NUMBER_F
```

Comment:
Use `mapping_parcels_field[self.names.OP_PARCEL_T_PARCEL_NUMBER_F]` directly, no need for more logic to get the key.

Reply:
Adjustment was made, thanks.